resources/testdata/glyphs2/WghtVar_AxisMappings.glyphs

```diff
@@ -0,0 +1,187 @@
+{
+.appVersion = "3148";
+DisplayStrings = (
+"-",
+"!"
+);
+customParameters = (
+{
+name = Axes;
+value = (
+{
+Name = Weight;
+Tag = wght;
+}
+);
+},
+{
+name = "Variable Font Origin";
+value = "medium";
+},
+{
+name = "Axis Mappings";
+value = {
+opsz = {
+12 = 12;
+72 = 72;
+};
+wght = {
+100 = 40;
+200 = 46;
+300 = 51;
+400 = 57;
+500 = 62;
+600 = 68;
+700 = 73;
+};
+};
+}
+);
+date = "2022-12-01 04:52:20 +0000";
+familyName = WghtVar;
+fontMaster = (
+{
+alignmentZones = (
+"{800, 16}",
+"{0, -16}",
+"{-200, -16}"
+);
+ascender = 800;
+capHeight = 0;
+descender = -200;
+id = m01;
+weightValue = 40;
+xHeight = 0;
+},
+{
+ascender = 800;
+capHeight = 500;
+descender = -200;
+id = "medium";
```

Comment:
I suspect you wrote this master id by hand? If Glyphs.app doesn't complain, I won't either :)
Usually these are strings in UUID format (`import uuid; uuid.uuid4()` in python)

Reply:
Yes, I manually edited it because reading a UUID was making my head hurt and we really don't care it's a UUID as far as I can tell